daemon/vendorplugin.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"context"
+	"net"
+	"runtime"
+
+	"github.com/go-logr/logr"
+	pb "github.com/openshift/dpu-operator/dpu-api/gen"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	VendorPluginSocketPath string = "/var/run/vendor-plugin.sock"
```

Comment:
Please put a FIXME to create a folder for dpu daemon within `/var/run`.

Reply:
I've moved it into daemon directory. Since the plugin is the server, it will create the directory.